ehrapy/api/tools/__init__.py

```diff
@@ -1,4 +1,6 @@
 from ehrapy.api.tools._hpo import HPO
-from ehrapy.api.tools._scanpy_tl_api import *  # noqa: E402,F403
 from ehrapy.api.tools.nlp._medcat import MedCAT
 from ehrapy.api.tools.nlp._translators import DeepL
+
+# TODO: uncomment when fa2 issue got fixed (see scanpy)
```

Comment:
No. This cannot be commented! We need this import. We need to restrict users to Python 3.8 and that's it for the moment.